contrastive_vi/model/total_contrastive_vi.py

```diff
@@ -369,6 +371,7 @@ def get_normalized_expression(
         Return type is ``pd.DataFrame`` unless ``return_numpy`` is True.
         """
         adata = self._validate_anndata(adata)
+        adata_manager = self.get_anndata_manager(adata)
```

Comment:
This + the remaining changes in this file simply make our normalized expression function compatible with scvi-tools 0.16 (currently it errors out).